docs/6-Troubleshooting/mysql-perm-error.md

```diff
@@ -0,0 +1,35 @@
+### `kool run mysql` or `kool run setup`
+
+**Problem:**
+
+> Access denied for root on mysql
```

Comment:
Same as previous comment...if it is not a mysql issue only, the text could be more generic